engine/inc/com/centreon/engine/service.hh

```diff
@@ -192,6 +193,8 @@ class service : public notifier {
   host const* get_host_ptr() const;
   host* get_host_ptr();
   bool get_host_problem_at_last_check() const;
+  tag_map& mut_tags() noexcept;
+  const tag_map& tags() const noexcept;
```

Comment:
service and host inherits from checkable.
So, tags would be better in checkable, then you won't write the same code twice...